database/schema.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose')
```

Comment:
Could we name this file something that is more illustrative about what the file is responsible for?  Models? Schema? 

As well, I think we need to look at strategies for connecting to the DB once in the application, instead of opening new connections in each module - please connect with @harmanlearns and @jessehall3 to discuss.